persistent/tests/test_list.py

```diff
@@ -268,13 +268,22 @@ def test_setslice(self):
         self.assertEqual(inst, [1, 2, 3])
         self.assertTrue(inst._p_changed)
 
-    def test_delslice(self):
+    def test_delslice_nonempty(self):
         inst = self._makeOne([1, 2, 3])
         self.assertFalse(inst._p_changed)
         self.assertEqual(inst, [1, 2, 3])
         del inst[:]
+        self.assertEqual(inst, [])
         self.assertTrue(inst._p_changed)
```

Comment:
We need a test here for a non-empty list with a non-"all" slice.

Reply:
Tests and fixes added in https://github.com/zopefoundation/persistent/pull/128